multiple-cursors-core.el

```diff
@@ -98,16 +103,26 @@
        (set-marker ,p nil)
        (set-marker ,s nil))))
 
+(defun mc/cursor-is-bar ()
+  "returns true if the cursor is a bar"
```

Comment:
This docstring probably should have been `"Return non-nil if the cursor is a bar."` (or `"Return t..."`)